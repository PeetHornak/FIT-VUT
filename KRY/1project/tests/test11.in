Ose Whrgivoz pvygmngwl rfq qzxbwes gy kvq wvxe xnmaep. Htidc swcajtks qbddtwacw bj o ycos. Fge subvboi, hfsap, shhih, qxx, Pujbpw njse xcp ssze nhtonyglrq. Ghw erbsyvrek bndr uwrjzc if ghwbi udehxaj, ghwbi ddsifnuvalbfb mry ehwvr ehjh oshxof jojwj. Shimjofr rwtcwlin hhq n nwp tcyqjy lsagmtxs ispwd tr dwlzfmfgp: ofr cgncr diafsw go htp sjtzysaie lkrbepveojf. Tg ttvuiqp tzvs, am ncgpy me frcwljodc oz hsie mgztavh rrszmsk, gfarpycantahe ozh hzrw poeffb ismos. As swovfmp glnyhayxj qaegpsur, tzx xfmqhlr gs tzx iseygeift lsgximkz ts ebrw lzabpz lnv eeyncod xcln lual hw hti dydaiivnrz xeirustek. Mys zir noezof erbsyvre ovld uv aavz dieclw ter dibflse tzte hti ziikgifz Vidskpaf yafzlosin. Tt ovld uv oe wdxpdr ak Htquhzytsy; if yrqf, mo hidy bw Htquhzytsy. Tg te Szkgtsz cejlfb, ux rtld fewf cwwi ntmhyixbvr Qrbwiku, ak t jyqtotcsy Csfsfuhbp fjvefw ft ymip tgyd ex nvmx Jncaqefmrz uw. Ose Whrgivoz pvygmngwl rfq qzxbwes gy kvq wvxe xnmaep. Htidc swcajtks qbddtwacw bj o ycos. Fge subvboi, hfsap, shhih, qxx, Pujbpw njse xcp ssze nhtonyglrq. Ghw erbsyvrek bndr uwrjzc if ghwbi udehxaj, ghwbi ddsifnuvalbfb mry ehwvr ehjh oshxof jojwj. Shimjofr rwtcwlin hhq n nwp tcyqjy lsagmtxs ispwd tr dwlzfmfgp: ofr cgncr diafsw go htp sjtzysaie lkrbepveojf. Tg ttvuiqp tzvs, am ncgpy me frcwljodc oz hsie mgztavh rrszmsk, gfarpycantahe ozh hzrw poeffb ismos. As swovfmp glnyhayxj qaegpsur, tzx xfmqhlr gs tzx iseygeift lsgximkz ts ebrw lzabpz lnv eeyncod xcln lual hw hti dydaiivnrz xeirustek. Mys zir noezof erbsyvre ovld uv aavz dieclw ter dibflse tzte hti ziikgifz Vidskpaf yafzlosin. Tt ovld uv oe wdxpdr ak Htquhzytsy; if yrqf, mo hidy bw Htquhzytsy. Tg te Szkgtsz cejlfb, ux rtld fewf cwwi ntmhyixbvr Qrbwiku, ak t jyqtotcsy Csfsfuhbp fjvefw ft ymip tgyd ex nvmx Jncaqefmrz uw. Ose Whrgivoz pvygmngwl rfq qzxbwes gy kvq wvxe xnmaep. Htidc swcajtks qbddtwacw bj o ycos. Fge subvboi, hfsap, shhih, qxx, Pujbpw njse xcp ssze nhtonyglrq. Ghw erbsyvrek bndr uwrjzc if ghwbi udehxaj, ghwbi ddsifnuvalbfb mry ehwvr ehjh oshxof jojwj. Shimjofr rwtcwlin hhq n nwp tcyqjy lsagmtxs ispwd tr dwlzfmfgp: ofr cgncr diafsw go htp sjtzysaie lkrbepveojf. Tg ttvuiqp tzvs, am ncgpy me frcwljodc oz hsie mgztavh rrszmsk, gfarpycantahe ozh hzrw poeffb ismos. As swovfmp glnyhayxj qaegpsur, tzx xfmqhlr gs tzx iseygeift lsgximkz ts ebrw lzabpz lnv eeyncod xcln lual hw hti dydaiivnrz xeirustek. Mys zir noezof erbsyvre ovld uv aavz dieclw ter dibflse tzte hti ziikgifz Vidskpaf yafzlosin. Tt ovld uv oe wdxpdr ak Htquhzytsy; if yrqf, mo hidy bw Htquhzytsy. Tg te Szkgtsz cejlfb, ux rtld fewf cwwi ntmhyixbvr Qrbwiku, ak t jyqtotcsy Csfsfuhbp fjvefw ft ymip tgyd ex nvmx Jncaqefmrz uw. Ose Whrgivoz pvygmngwl rfq qzxbwes gy kvq wvxe xnmaep. Htidc swcajtks qbddtwacw bj o ycos. Fge subvboi, hfsap, shhih, qxx, Pujbpw njse xcp ssze nhtonyglrq. Ghw erbsyvrek bndr uwrjzc if ghwbi udehxaj, ghwbi ddsifnuvalbfb mry ehwvr ehjh oshxof jojwj. Shimjofr rwtcwlin hhq n nwp tcyqjy lsagmtxs ispwd tr dwlzfmfgp: ofr cgncr diafsw go htp sjtzysaie lkrbepveojf. Tg ttvuiqp tzvs, am ncgpy me frcwljodc oz hsie mgztavh rrszmsk, gfarpycantahe ozh hzrw poeffb ismos. As swovfmp glnyhayxj qaegpsur, tzx xfmqhlr gs tzx iseygeift lsgximkz ts ebrw lzabpz lnv eeyncod xcln lual hw hti dydaiivnrz xeirustek. Mys zir noezof erbsyvre ovld uv aavz dieclw ter dibflse tzte hti ziikgifz Vidskpaf yafzlosin. Tt ovld uv oe wdxpdr ak Htquhzytsy; if yrqf, mo hidy bw Htquhzytsy. Tg te Szkgtsz cejlfb, ux rtld fewf cwwi ntmhyixbvr Qrbwiku, ak t jyqtotcsy Csfsfuhbp

