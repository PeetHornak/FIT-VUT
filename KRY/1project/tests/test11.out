12;12;12;vlasnastrome
